doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -111,8 +109,9 @@ for a single-pod RayCluster will a log-processing sidecar.
 :language: yaml
 ```
 
-### Deploying a RayCluster with logging CR
 (kuberay-logging-tldr)=
+### Deploying a RayCluster with logging CR
```

Comment:
Do we need to spell out CR to avoid confusion?

Reply:
Not sure. cc: @kevin85421

Reply:
I believe we want to use "sidecar" instead of "CR"